src/controller/cpp/Makefile

```diff
@@ -102,12 +115,14 @@ $(LIBCONTROLLER):
 -include $(DEPENDENCIES)
 
 # cpp compilation rule
-%.o:%.cpp
+$(OBJDIR)/%.o:%.cpp
+	@echo "# compiling "$<
```

Comment:
It seems that on linux $(OBJDIR) has an additional slash (`build//Accelerometer.o`), here is the log:
```
# compiling Accelerometer.cpp
g++ -c -fpic -Wall -I/home/stefi/develop/webots/include/controller/c -I/home/stefi/develop/webots/include/controller/cpp -o build//Accelerometer.o Accelerometer.cpp
```

But it works correctly.

Reply:
Again, you should write "make release" to avoid this problem...

Reply:
It is fine now that `make` without target just prints the message and doesn't compile.
Otherwise, it `make` was triggering the compilation, all the path should have be correctly also without target.